apps/meteor/tests/e2e/omnichannel/omnichannel-livechat-logo.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { IS_EE } from '../config/constants';
+import { createAuxContext } from '../fixtures/createAuxContext';
+import { Users } from '../fixtures/userStates';
+import { OmnichannelLiveChat, OmnichannelSettings } from '../page-objects';
+import { test, expect } from '../utils/test';
+
+test.skip(!IS_EE, 'Enterprise Only');
+
+test.use({ storageState: Users.admin.state });
+
+test.describe('OC - Livechat - Widget logo', async () => {
+	let poLiveChat: OmnichannelLiveChat;
+	let poOmnichannelSettings: OmnichannelSettings;
+
+	test.beforeEach(async ({ browser, api }) => {
+		const { page: livechatPage } = await createAuxContext(browser, Users.user1, '/livechat', false);
+
+		poLiveChat = new OmnichannelLiveChat(livechatPage, api);
+	});
+
+	test.beforeEach(async ({ page }) => {
+		poOmnichannelSettings = new OmnichannelSettings(page);
+
+		await page.goto('/admin/settings/Omnichannel');
+	});
+
+	test.afterAll(async ({ api }) => {
+		const res = await api.post('/assets.unsetAsset', { assetName: 'livechat_widget_logo' });
+		if (res.status() !== 200) {
+			throw new Error('Failed to unset asset');
+		}
+	});
+
+	test('OC - Livechat - Change widget logo', async () => {
+		await test.step('expect to have default logo', async () => {
+			await poOmnichannelSettings.group('Livechat').click();
+			await expect(poLiveChat.imgLogo).not.toBeVisible();
+			await expect(poOmnichannelSettings.imgLivechatLogoPreview).not.toBeVisible();
+		});
+
+		await test.step('expect to change widget logo', async () => {
+			await expect(poOmnichannelSettings.labelLivechatLogo).toHaveText('Livechat widget logo (svg, png, jpg)');
+			await poOmnichannelSettings.inputLivechatLogo.setInputFiles('./tests/e2e/fixtures/files/test-image.jpeg');
+			await expect(poOmnichannelSettings.imgLivechatLogoPreview).toBeVisible();
+
+			await poLiveChat.page.reload();
```

Comment:
Since you are already doing a reload here, I would suggest breaking up those steps into separate tests, each  loading the page after the asset was changed, I believe it would be faster than consecutive reloads

Reply:
@MartinSchoeler I don't think this idea would work very well for these types of tests. Since I want to see multiple changes made to the setting reflected on the widget, creating a separate session every time does not seem very efficient 🤔